lib/cloud-watch-event-listener.js

```diff
@@ -156,31 +159,27 @@ class CloudWatchEventListener extends events.EventEmitter {
           workerType: workerType,
         });
       }
+      let opts = {
+        workerType,
+        region,
+        az,
+        instanceType,
+        imageId,
+        id,
+        state,
+        srid: srid ? srid : 'undefined',
```

Comment:
I prefer `srid || 'undefined'` in this sort of case generally. Also maybe using `'undefined'` the string is a bit confusing since it is a concept in js? Both are just nits.